selfdrive/common/visionipc.c

```diff
@@ -119,19 +106,17 @@ int visionstream_init(VisionStream *s, VisionStreamType type, bool tbuffer, Visi
   return 0;
 }
 
-void visionstream_release(VisionStream *s) {
-  int err;
-  if (s->last_idx >= 0) {
-    VisionPacket rep = {
-      .type = VIPC_STREAM_RELEASE,
-      .d = { .stream_rel = {
-        .type = s->last_type,
-        .idx = s->last_idx,
-      }}
-    };
-    err = vipc_send(s->ipc_fd, &rep);
-    s->last_idx = -1;
-  }
+static int visionstream_release(VisionStream *s) {
+  VisionPacket rep = {
```

Comment:
why remove the check on last_idx here instead of the places where visionstream_release is called?